docs/sample_sheet_schema/schema_specification.md

```diff
@@ -102,7 +103,7 @@ Whether or not the field should contain a unique value over the entire sample sh
 
 Default: `false`
 
-- Can be `true`, in which case the field should be unique in the this column for all samples
+- Can be `true`, in which case all fields in this column should be unique for all samples.
```

Comment:
I think throughout most of this documentation, fields = columns.

Do you mean rows, or samples?

```suggestion
- Can be `true`, in which case the value for this field should be unique for all samples in the sample sheet.
```

Reply:
Good point, thanks for the consistency in definitions here :)